README.md

```diff
@@ -121,5 +117,5 @@ To build the application:
 # Using node v4.2 or greater
 $ npm install
 $ npm run build
-$ ./bin/tick --help
-```
```

Comment:
Ah, habit, sorry. It's `EOF`, which terminates the section of code.
Fixed to match project preference